src/main/java/com/projecty/projectyweb/task/TaskController.java

```diff
@@ -83,10 +83,10 @@ public void addTaskPost(
         return map;
     }
 
-    @PostMapping("deleteTask")
+    @PostMapping({"deleteTask/project/{projectId}/task/{taskId}", "deleteTask/task/{taskId}", "deleteTask/project/task/{taskId}"})
```

Comment:
This can even be a DELETE action with a `@DeleteMapping`

Reply:
I changed @PostMapping to @DeleteMapping for this method and replaced left @RequestParam with @PathVariable